js/samples/04.ai-apps/h.datasource-azureOpenAI/README.md

```diff
@@ -31,20 +31,15 @@ This guide will show you have the set up your custom copilot for Teams using Azu
 | ----------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [Visual Studio Code](https://code.visualstudio.com/download)                                                                  | Typescript build environments. Use the latest version.                                                                                                                                                                                                              |
 | [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) (5.3.x or greater) | Microsoft Visual Studio Code extension that creates a project scaffolding for your app. Use the latest version.                                                                                                                                                     |
-| [Git](https://git-scm.com/downloads)                                                                                          | Git is a version control system that helps you manage different versions of code within a repository.                                                                                                                                                               |
```

Comment:
I don't think we want to remove this part. If people are cloning the repo, they will still need git.

Reply:
I was trying to simplify this and thought it was more of a recommendation as the Teams AI QuickStart guide did not have it. 

Reference: https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/teams%20conversational%20ai/conversation-ai-quick-start?tabs=OpenAI-key%2COpenAI-key2&pivots=qs-javascript